src/mock/v1.2/channelService-1_2.ts

```diff
@@ -65,7 +65,7 @@ export class Fdc3CommandExecutor1_2 {
   //get broadcast service and broadcast the given context type
   async broadcastContextItem(contextType, channel, historyItems, testId) {
     let broadcastService = this.getBroadcastService(channel.type);
-    broadcastService.broadcast(contextType, historyItems, channel, testId);
```

Comment:
I thought we'd already rejected doing this once - its a synchronous call in 1.2

Waits were added before windows were closed to avoid the need for this.

Reply:
@ksgeorgieva : 

@Joe-Dunleavy and I have just removed these `await`s and `async`s because they are redundant.  Could you check out the latest code and retest please?  

Given that the methods involved _don't_ return promises, this should have no effect at all.

thanks!